.github/workflows/main.yml

```diff
@@ -0,0 +1,500 @@
+          mv coverage coverage-${{ matrix.neo4j-version }}
+        working-directory: packages/${{ matrix.package }}
+        env:
+          NEO_USER: neo4j
+          NEO_PASSWORD: testpassword
+          NEO_URL: bolt://localhost:7687
+      - name: Archive coverage report
+        uses: actions/upload-artifact@v2
+        with:
+          name: integration-coverage-${{ matrix.package }}
+          path: packages/${{ matrix.package }}/coverage-${{ matrix.neo4j-version }}/
+
+  aura-teardown:
+    needs:
+      - cla-check
+      - cache-dependencies
+    if: |
+      always() &&
+      (needs.cla-check.result == 'success' || needs.cla-check.result == 'skipped') &&
+      (needs.cache-dependencies.result == 'success')
+
+    strategy:
+      matrix:
+        aura-instance:
+          - free
+          - professional
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: lts/*
+          cache: yarn
+      - name: Install dependencies
+        run: yarn --immutable
+      - name: Install ts-node and typescript
+        run: npm install -g ts-node typescript
+      - name: Uppercase instance name
+        run: |
+          lowercase_instance=${{ matrix.aura-instance }}
+          echo "AURA_URI_SECRET=AURA_${lowercase_instance^^}_URI" >>${GITHUB_ENV}
+          echo "AURA_PASSWORD_SECRET=AURA_${lowercase_instance^^}_PASSWORD" >>${GITHUB_ENV}
+      - name: Delete all data in instance
+        run: ts-node tests/integration/teardown.ts
+        working-directory: packages/graphql
+        env:
+          NEO_USER: neo4j
+          NEO_PASSWORD: ${{ secrets[env.AURA_PASSWORD_SECRET] }}
+          NEO_URL: ${{ secrets[env.AURA_URI_SECRET] }}
+
+  integration-tests-aura:
+    needs: aura-teardown
+    if: |
+      always() &&
+      (needs.aura-teardown.result == 'success')
+
+    strategy:
+      matrix:
+        package:
+          - graphql
+          - ogm
+        aura-instance:
+          - free
+          - professional
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: lts/*
+          cache: yarn
+      - name: Install dependencies
+        run: yarn --immutable
+      - name: Uppercase instance name
+        run: |
+          lowercase_instance=${{ matrix.aura-instance }}
+          echo "AURA_URI_SECRET=AURA_${lowercase_instance^^}_URI" >>${GITHUB_ENV}
+          echo "AURA_PASSWORD_SECRET=AURA_${lowercase_instance^^}_PASSWORD" >>${GITHUB_ENV}
+      - name: Run @neo4j/graphql integration tests
+        run: |
+          yarn test:int -t '^(?!delete should delete a movie, a single nested actor and another movie they act in|update should delete a nested actor and one of their nested movies, within an update block)' --coverage
```

Comment:
What's going on with this line? Why are we calling out specific tests?

Reply:
There is a bug in the current Aura drop right now - fixed in the next drop and this can be removed when that is released. 🙂